tools/src/symbol_graph_edge_kind.rs

```diff
@@ -0,0 +1,12 @@
+#[derive(Clone)]
+pub enum EdgeKind {
+    Default, // solid line, closed arrow ("normal")
+    // These value are meant to be UML-ish
+    Inheritance,    // solid line, open arrow ("onormal")
+    Implementation, // dashed line, open arrow ("onormal")
+    Composition,    // solid line, closed diamond ("diamond")
+    Aggregation,    // solid line, open diamond ("odiamond")
+    // These are more specific searchfox concepts
+    IPC,           // dotted line, weird vee arrow ("vee")
+    CrossLanguage, // JNI-like; solid line, left-half-closed arrow ("lnormal")
+}
```

Comment:
This is moved from `cmd_pipeline/symbol_graph.rs`,
in order to remove the dependency from `ontology_mapping.rs` to the remaining of `cmd_pipeline/symbol_graph.rs`.

Reply:
Makes sense, thanks!  That whole cluster of code has evolved in a somewhat haphazard way and likely could benefit from a good spring cleaning! ;)